pkgs/applications/networking/cluster/kubevela/default.nix

```diff
@@ -0,0 +1,64 @@
+{ buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, lib
+, stdenv
+, testers
+, kubevela
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "kubevela";
+  version = "1.9.7";
+
+  src = fetchFromGitHub {
+    owner = "kubevela";
+    repo = "kubevela";
+    rev = "v${version}";
+    hash = "sha256-vkq4XuSE+DeCMRBR7Z276TJIpOi0qEop7FBD2PPiQN4=";
+  };
+
+  vendorHash = "sha256-rtzPrbjIhxDUcOZOht0PKidCacvTYUNeNUbisgKKR2A=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/oam-dev/kubevela/version.VelaVersion=${version}"
```

Comment:
Maybe add "github.com/oam-dev/kubevela/version.GitRevision=undefined" like https://github.com/kubevela/kubevela/blob/62efa9c78744070aa3cbdf097d4440610f15ffb9/Dockerfile#L30 too?

Reply:
First, thanks for the review.
I could add it but `GitRevision=undefined` does not seem to make a meaningful difference.

Reply:
Does not seems to have any code reference too.